app/javascript/styles/_brand.scss

```diff
@@ -88,6 +88,10 @@
   font-size: 24px;
 }
 
+figcaption {
```

Comment:
can you add a comment that this is related to CKeditor?   and maybe move it to _global with the other ckeditor specific stuff?